packages/circuit-ui/components/Table/Table.tsx

```diff
@@ -288,6 +303,17 @@ class Table extends Component<TableProps, TableState> {
     this.updateSort(i, nextDirection, sortedRows);
   };
 
+  getRows = (
+    rows: Row[],
+    sortDirection?: Direction | undefined,
+    sortedRow?: number | undefined,
```

Comment:
Should we also rename these for clarity?
```suggestion
    initialSortDirection?: Direction | undefined,
    initialSortedRow?: number | undefined,
```

We could also make the method itself `getInitialRows` if it's only for the initial render

Reply:
good point. I will rename the function.